_sass/components/_intro.scss

```diff
@@ -20,12 +20,13 @@
     color: $white;
     font-size: 16px;
     font-weight: 400;
-    letter-spacing: 0.01em;
+    letter-spacing: .01em;
     line-height: 26px;
   }
 
   #content {
     background: $primary;
+    opacity: .9
```

Comment:
Okay with me, but... we actually had designers intentionally designed this to be opaque. 

https://www.figma.com/file/7uczsKDoFUxT5nzLrwC9aI/2023-ZGZG%E6%98%A5%E6%99%9A%E7%BD%91%E7%AB%99%E7%BB%84?node-id=1%3A2

Reply:
Requested access. It seems we have no designers this year yet. 
Revised according to Lizzi and Elise's suggestions.